requirements/common_constraints.txt

```diff
@@ -16,7 +16,7 @@
 #  this file from Github directly. It does not require packaging in edx-lint.
 
 # using LTS django version
```

Comment:
I wonder if we should use https://github.com/openedx/edx-lint/blob/master/edx_lint/files/common_constraints.txt instead of maintaining this file?

Reply:
That's my end-goal is to correct that file but I wanted to get this landed in edx-platform because a bunch of other requirements upgrades have been blocked behind this.  PR incoming over there shortly.

Reply:
[inform] There exists a ticket and proposal for how to handle repo-specific overrides of common constraints in the future here: https://github.com/openedx/edx-lint/issues/355#issuecomment-1634634936.

Reply:
PR for edx-lint is here: https://github.com/openedx/edx-lint/pull/500